js/common.js

```diff
@@ -0,0 +1,121 @@
+
+const dropdownWrapper = document.querySelector('.dropdown-wrapper');
+const dropdownHeader = document.querySelector('.dropdown-header');
+const dropdownBody = document.querySelector('.dropdown-body');
+const dropdownIcon = document.querySelector('.dropdown-header .icon');
+
+const toggleDropdown = () => {
+    dropdownWrapper.classList.toggle('open');
+}
+
+const closeDropdownOnClickOutside = (event) => {
+    if (!dropdownWrapper.contains(event.target)) {
+        dropdownWrapper.classList.remove('open');
+    }
+}
+
+async function filterCountriesByRegion(region) {
+    try {
+        console.log('Filtering by region:', region);
+
```

Comment:
Remove the console log, its fine when developing a feature but not when pushing to the main branch